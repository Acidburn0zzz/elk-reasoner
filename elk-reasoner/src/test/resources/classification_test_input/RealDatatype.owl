Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(:=<http://www.semanticweb.org/Test.owl#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)

Ontology(<http://www.semanticweb.org/Test.owl>

EquivalentClasses(:Real_Larger_Or_Equal_Than_10 DataSomeValuesFrom(:dp DatatypeRestriction(owl:real xsd:minInclusive "10"^^xsd:integer)))
EquivalentClasses(:Real_Larger_Than_10 		DataSomeValuesFrom(:dp DatatypeRestriction(owl:real xsd:minExclusive "10"^^xsd:integer)))
EquivalentClasses(:Real_Less_Or_Equal_Than_10 	DataSomeValuesFrom(:dp DatatypeRestriction(owl:real xsd:maxInclusive "10"^^xsd:integer)))
EquivalentClasses(:Real_Less_Than_10		DataSomeValuesFrom(:dp DatatypeRestriction(owl:real xsd:maxExclusive "10"^^xsd:integer)))

EquivalentClasses(:Reals_Between_0_and_10_Ex 	DataSomeValuesFrom(:dp DatatypeRestriction(owl:real xsd:minExclusive "0"^^xsd:integer xsd:maxExclusive "10"^^xsd:integer)))
EquivalentClasses(:Reals_Between_0_and_10_Inc 	DataSomeValuesFrom(:dp DatatypeRestriction(owl:real xsd:minInclusive "0"^^xsd:integer xsd:maxInclusive "10"^^xsd:integer)))
EquivalentClasses(:Reals_Between_0_and_10_IncEx DataSomeValuesFrom(:dp DatatypeRestriction(owl:real xsd:minInclusive "0"^^xsd:integer xsd:maxExclusive "10"^^xsd:integer)))
EquivalentClasses(:Reals_Between_0_and_10_ExInc DataSomeValuesFrom(:dp DatatypeRestriction(owl:real xsd:minExclusive "0"^^xsd:integer xsd:maxInclusive "10"^^xsd:integer)))

EquivalentClasses(:Reals_Between_1_and_12_Ex	DataSomeValuesFrom(:dp DatatypeRestriction(owl:real xsd:minExclusive "1"^^xsd:integer xsd:maxExclusive "12"^^xsd:integer)))
EquivalentClasses(:Reals_Between_1_and_12_Inc	DataSomeValuesFrom(:dp DatatypeRestriction(owl:real xsd:minInclusive "1"^^xsd:integer xsd:maxInclusive "12"^^xsd:integer)))
EquivalentClasses(:Reals_Between_1_and_12_ExInc	DataSomeValuesFrom(:dp DatatypeRestriction(owl:real xsd:minExclusive "1"^^xsd:integer xsd:maxInclusive "12"^^xsd:integer)))
EquivalentClasses(:Reals_Between_1_and_12_IncEx	DataSomeValuesFrom(:dp DatatypeRestriction(owl:real xsd:minInclusive "1"^^xsd:integer xsd:maxExclusive "12"^^xsd:integer)))

SubClassOf(:InconsistentReal			DataSomeValuesFrom(:dp DatatypeRestriction(owl:real xsd:minExclusive "10"^^xsd:integer xsd:maxExclusive "0"^^xsd:integer)))
)
