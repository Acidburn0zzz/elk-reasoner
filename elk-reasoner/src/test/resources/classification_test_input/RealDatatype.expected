Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(:=<http://www.semanticweb.org/Test.owl#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)

Ontology(<http://www.semanticweb.org/Test.owl>

SubClassOf(:Real_Larger_Than_10			:Real_Larger_Or_Equal_Than_10)
SubClassOf(:Real_Less_Than_10			:Real_Less_Or_Equal_Than_10)
SubClassOf(:Reals_Between_0_and_10_Ex		:Reals_Between_0_and_10_ExInc)
SubClassOf(:Reals_Between_0_and_10_Ex		:Reals_Between_0_and_10_IncEx)
SubClassOf(:Reals_Between_0_and_10_ExInc	:Reals_Between_0_and_10_Inc)
SubClassOf(:Reals_Between_0_and_10_Inc		:Real_Less_Or_Equal_Than_10)
SubClassOf(:Reals_Between_0_and_10_IncEx	:Real_Less_Than_10)
SubClassOf(:Reals_Between_0_and_10_IncEx	:Reals_Between_0_and_10_Inc)
SubClassOf(:Reals_Between_1_and_12_Ex		:Reals_Between_1_and_12_ExInc)
SubClassOf(:Reals_Between_1_and_12_Ex		:Reals_Between_1_and_12_IncEx)
SubClassOf(:Reals_Between_1_and_12_ExInc	:Reals_Between_1_and_12_Inc)
SubClassOf(:Reals_Between_1_and_12_IncEx	:Reals_Between_1_and_12_Inc)

EquivalentClasses(owl:Nothing :InconsistentReal)
)
